### PR TITLE
Addon for Citrix Ingress

### PR DESCRIPTION
**Pull request by christus02**
**Addon for Citrix Ingress Controller**

Support for [Citrix Ingress Controller](https://github.com/citrix/citrix-k8s-ingress-controller) to be deployed as an Ingress using KOPS addon.

### CODE REVIEW

**Comment by christus02** [comment]
"I signed it"

File: addons/ingress-citrix/v1.1.1.yaml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cpx-ingress-k8s-role
+  namespace: default
+
+---
+
+apiVersion: extensions/v1beta1
```
Comment (zetaab) [inline addons/ingress-citrix/v1.1.1.yaml]: could you please use `apps/v1` instead of this beta which is no longer supported in newest k8s

**Comment by christus02** [comment]
Could you please merge this PR?

**Comment by zetaab** [comment]
I do not have access to do that 

/assign @chrisz100

**Pull request merged**